Kochbuch/src/app/app.component.html

```diff
@@ -1,4 +1,214 @@
+              <div class="col-lg-2">
+                <input type="checkbox" name="backen" id="backen" autocomplete="off"><label for="backen">Backen</label>
+              </div>
+              <div class="col-lg-2">
+                <input type="checkbox" name="cocktails" id="cocktails" autocomplete="off"><label for="cocktails">Cocktails</label>
+              </div>
+              <div class="col-lg-2">
+                <input type="checkbox" name="desserts" id="desserts" autocomplete="off"><label for="desserts">Desserts</label>
+              </div>
+            </div>
+            <br>
+            <div class="row">
+              <div class="col-lg-2">
+                <input type="checkbox" name="vegetraisch" id="vegetraisch" autocomplete="off"><label for="vegetraisch">vegetraisch</label>
+              </div>
+              <div class="col-lg-2">
+                <input type="checkbox" name="vegan" id="vegan" autocomplete="off"><label for="vegan">vegan</label>
+              </div>
+              <div class="col-lg-2">
+                <input type="checkbox" name="bio" id="bio" autocomplete="off"><label for="bio">bio</label>
+              </div>
+              <div class="col-lg-3">
+                <input type="checkbox" name="kalorienarm" id="kalorienarm" autocomplete="off"><label for="kalorienarm">kalorienarm</label>
+              </div>
+            </div>
+            <br>
+
+            <label>Email</label>
+            <div class="row">
+              <div class="col-lg-10">
+                <div class="input-group">
+                  <input type="email" id="email" name="email" placeholder="E-Mail" class="form-control"/>
+                </div>
+              </div>
+            </div>
+            <br>
+
+            <label>Benutzername</label>
+            <div class="row">
+              <div class="col-lg-10">
+                <div class="input-group">
+                  <input type="text" id="user" name="user" placeholder="Benutzername" class="form-control"/>
+                </div>
+              </div>
+            </div>
+            <br>
+
+            <label>Passwort</label>
+            <div class="row">
+              <div class="col-lg-10">
+                <div class="input-group">
+                  <input type="password" id="password1" name="password1" placeholder="Passwort" class="form-control"/>
+                </div>
+              </div>
+            </div>
+            <br>
+
+            <div class="row">
+              <div class="col-lg-10">
+                <div class="input-group">
+                  <input type="password" id="password2" name="password2" placeholder="Passwort wiederholen"
+                         class="form-control"/>
+                </div>
+              </div>
+            </div>
+            <br>
+            <label>Nutzungsbedingungen</label>
+            <div class="row">
+              <div class="form-inline col-lg-10">
+                <div class="checkbox">
```

Comment:
die Checkboxklasse wird nicht in der CSS verwendet, also sollte die Bootstrapklasse dafür verwendet werden
http://getbootstrap.com/docs/4.0/components/forms/#checkboxes-and-radios